src/main/xerus/monstercat/api/Covers.kt

```diff
@@ -16,13 +16,16 @@ object Covers {
 	private fun coverCacheFile(coverUrl: String, size: Int): File {
 		coverCacheDir.mkdirs()
 		val newFile = coverCacheDir.resolve(coverUrl.substringBeforeLast('/').substringAfterLast('/').replaceIllegalFileChars())
```

Comment:
If the name of the coverUrl has changed, can't we simplify this? Otherwise, using `split` seems more sensible to me than two substring operations.

Reply:
Also, please write a test for this method. Should be easy.

Reply:
I have not a single idea on how to write tests unfortunately.

Reply:
Then look into the existing ones :)

Reply:
Split usage 0f20221b01975b546f91c6020c4504f9872bbceb